packages/wallet-specs/src/ChannelStoreEntry.ts

```diff
@@ -79,30 +73,28 @@ export class ChannelStoreEntry implements IChannelStoreEntry {
       privateKey,
       participants,
       channel,
-      funding,
+      funding
     }: IChannelStoreEntry = this;
     return {
       supportedState,
       unsupportedStates,
       privateKey,
       participants,
       channel,
-      funding,
```

Comment:
I like my trailing commas. Not having them is really annoying in a few cases.

Reply:
I like trailing commas too. I'm in favour of turning them on in `packages/wallet-specs/.prettierrc.js` and eventually for all of the packages.